test/e2e/specs/specs-wpcom/wp-editor__revisions.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * @group gutenberg
+ */
```

Comment:
Can we add the `calypso-pr` group too? This revision modal is actually added by Calypso, and is not part of Core!

Reply:
Added in 34af05384435710ef0a1e8e95a2b4916646084da.